.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,26 @@
+{
+	"name": " Codespace",
+	"image": "mcr.microsoft.com/vscode/devcontainers/javascript-node:0.202.3-16",
+	"settings": {
+		"workbench.colorTheme": "Default Dark+"
+	},
+	"extensions": [
+		"eamodio.gitlens",
+		"cschleiden.vscode-github-actions",
+		"redhat.vscode-yaml",
+		"bierner.markdown-preview-github-styles",
```

Comment:
Should we be using the extension pack for full github markdown support? [`bierner.github-markdown-preview`](https://marketplace.visualstudio.com/items?itemName=bierner.github-markdown-preview)

Reply:
It is not required and we haven't included it, because for the basic purposes of GitHub documentation writing in VS Code it is sufficient.

The `bierner.github-markdown-preview` adds a few minor things, which if you find useful, we will add them 🙂